src/Config/Level/TypeDeclarationLevel.php

```diff
@@ -77,7 +74,6 @@ final class TypeDeclarationLevel
         ReturnTypeFromStrictScalarReturnExprRector::class,
         ReturnTypeFromReturnDirectArrayRector::class,
         ReturnTypeFromReturnNewRector::class,
-        AddClosureReturnTypeFromReturnCastRector::class,
```

Comment:
I think existing closure rule can be kept, just add new one for general type for "other cases"

Reply:
Those are merged into one. Goal of this PR is to have single rule to handle closures.